database/src/main/kotlin/no/nav/su/se/bakover/database/beregning/PersistertMerknad.kt

```diff
@@ -0,0 +1,128 @@
+package no.nav.su.se.bakover.database.beregning
+
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import no.nav.su.se.bakover.domain.beregning.Merknad
+import java.time.LocalDate
+
+internal sealed class PersistertMerknad {
+
+    @JsonTypeInfo(
+        use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.PROPERTY,
+        property = "type",
+    )
+    @JsonSubTypes(
+        JsonSubTypes.Type(
+            value = Beregning.EndringGrunnbeløp::class,
```

Comment:
Har vi kun reguleringer knyttet til grunnbeløp? Er det ikke noen egne greier på ektefelles inntekt?

Reply:
Ikke helt sikker på hva du tenker på når du sier "reguleringer", men antar at du tenker på f.eks garantipensjonsnivå (styrer grensen for hvor mye eps kan tjene før vi begynner avkorting i enkelte tilfeller) - vi har muligheten til å legge til merknader som opplyser om denne typen endringer også.

Reply:
Med reguleringer tenker jeg på satsreguleringer i en eller annen forstand som vil føre til en utmålingsdifferanse (sumYtelse) mellom 2 beregner med helt likt grunnlag, men med forskjellig satser.

Men jeg tror ikke jeg klarer å rettferdiggjøre ovennevnte som merknader på en beregning tror jeg.